assets/lib/dialog/dialog.js

```diff
@@ -374,6 +374,15 @@
 				},
 			};
 
+			if (window.elementor &&
+				'elementor-template-library-modal' === userSettings?.id &&
+				'contact-buttons' === elementor?.config?.document?.type
+			) {
+				settings.hide.onOutsideClick = elementor?.config?.document?.panel?.allow_closing_remote_library ?? true;
+				settings.hide.onBackgroundClick = elementor?.config?.document?.panel?.allow_closing_remote_library ?? true;
+				settings.hide.onEscKeyPress = elementor?.config?.document?.panel?.allow_closing_remote_library ?? true;
+			}
```

Comment:
@nicoladj77 Is there another way to add these settings? AFAIU we are not allowed to edit anything under `lib` directly.

Reply:
@louiswol94 ok let me try it